promitor-agent-scraper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 version: 2.0.0
-appVersion: 2.0.0
+appVersion: 2.1.0
```

Comment:
Can you revert this please?

Reply:
Oops wrong parameter, sorry

Reply:
Reverted and bumped `version` instead

Reply:
Is that OK or should I leave the versions exactly as they were before?